javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Javascript file for Hack Your Future, Javascript1, homework Week4
+ * program will return a response that corresponds to the command.
+*/
+
+// Voice assistant
+
+let userName = "";
+let toDo = [];
+
+function getReply(command) {
+
+    if (typeof command !== "string") {
+        return "Please retype your Question";
+    }
+    let str = command.split(" ");
```

Comment:
`command.split()` returns an array, so the name `str` is confusing. Use variable names that explain what is in the variable. E.g. splitString, stringPieces.
```suggestion
    let stringPieces = command.split(" ");
```

Reply:
done.thanks